package.json

```diff
@@ -156,6 +157,30 @@
                                     "${workspaceFolder}"
                                 ]
                             },
+                            "componentLibrariesPort": {
+                                "type": "number",
+                                "description": "Port to access component libraries.",
+                                "default": 8080
+                            },
+                            "componentLibrariesOutDir": {
+                                "type": "string",
+                                "description": "Output folder the component libraries will be hosted in.",
+                                "default": "${workspaceFolder}/libs"
+                            },
+                            "componentLibraries": {
+                                "type": "array",
```

Comment:
Sorry for not catching this earlier. Using type "array" isn't good enough here, because the user won't get any intellisense for `rootDir`, `outFile`, `files`, etc. You should copy the `items` section for `files` down below into this. Then test and verify that in vscode, it provides intellisense for each of those items.